include/picongpu/particles/shapes/NGP.hpp

```diff
@@ -76,7 +76,7 @@ namespace picongpu
                     /** form factor of this particle shape.
```

Comment:
Why isn't this shape extended with the new interface?
For `Counter` i understand it would never be used, as this is not a "real" macroparticle shape

Reply:
I thought it is a shape that is not allowed to be used as particle shape.